Toolset/palettes/inspector/editors/com.livecode.pi.file.behavior.livecodescript

```diff
@@ -56,6 +56,13 @@ on mouseUp pButton
       put the result into tResult
       
       if there is a file tResult then
+         if the cImageFileRelative of stack "revPreferences" is not false then
+            local tStackPath, tFilePath
+            set the itemDelimiter to "/"
+            put item 1 to -2 of the filename of stack revTargetStack(line 1 of (the selObj)) into tStackPath
```

Comment:
There is a problem here in that the PI may not be operating on one of the selected objects. Indeed there may be no selected objects in which case there will be an error thrown on this line. Try `the cSelectedObjects of this stack`